.eslintrc.js

```diff
@@ -0,0 +1,36 @@
+module.exports = {
+  root: true,
+
+  env: {
+    node: true,
+  },
+
+  extends: [
+    'plugin:vue/recommended',
+    '@vue/airbnb',
+    '@vue/typescript',
+    'plugin:vue/recommended',
+    '@vue/typescript/recommended',
+  ],
+
+  plugins: [
+    'vuetify',
+  ],
+
+  rules: {
+    'no-console': process.env.NODE_ENV === 'production' ? 'warn' : 'off',
+    'no-debugger': process.env.NODE_ENV === 'production' ? 'warn' : 'off',
```

Comment:
Maybe these should raise errors instead of warnings. I can't remember if we treat warnings as errors in our build step.

Reply:
Agreed. I've updated it in cc3210b